src/main/resources/config/application.yaml

```diff
@@ -12,7 +12,7 @@ spring:
         consumeStudyUpdate-in-0:
           destination: ${powsybl-ws.rabbitmq.destination.prefix:}study.update
           group: directoryStudyUpdateGroup
-      source: publishDirectoryUpdate
+      output-bindings: publishDirectoryUpdate
```

Comment:
why did we change this ?

Reply:
source is deprecated so I replaced it with the correct one. It's not related to this PR really but I allowed myself to do it here because I know it won't cause a regression (I changed these annotation in all server in my spring-boot update PR but apparently forgot this one). If you still don't want it there, I will happily revert it.